amostra/client/commands.py

```diff
@@ -24,24 +24,47 @@ def __init__(self, sample_list=[], host=conf.conn_config['host'],
             Port tornado server runs on
         """
         self.host = host
-        self.port = port        
-        self._server_path = 'http://{}:{}/' .format(self.host, self.port)
-        if not isinstance(sample_list, list):
-            raise TypeError("sample_list must be a list")
-        if not isinstance(sample_list, list):
-            raise TypeError('Not a correct type for the constructor.Expects list')
-        self._sample_list = [dict(d) for d in sample_list]
-        ln = len(self._sample_list)
-        if ln != len(set(d['name'] for d in self._sample_list)):
-            raise ValueError("duplicate names")
-        if ln != len(set(d['uid'] for d in self._sample_list)):
-            raise ValueError("duplicate uids")
-        if sample_list:
-            domt = ujson.dumps(self._sample_list)
-            r = requests.post(self._server_path + 'sample',
-                              data=domt)
-            r.raise_for_status()
+        self.port = port
+#        if not isinstance(sample_list, list):
+#            raise TypeError("sample_list must be a list")
+#        if not isinstance(sample_list, list):
+#            raise TypeError('Not a correct type for the constructor.Expects list')
+#        self._sample_list = [dict(d) for d in sample_list]
+#        ln = len(self._sample_list)
+#        if ln != len(set(d['name'] for d in self._sample_list)):
+#            raise ValueError("duplicate names")
+#        if ln != len(set(d['uid'] for d in self._sample_list)):
+#            raise ValueError("duplicate uids")
+#        if sample_list:
+#            domt = ujson.dumps(self._sample_list)
+#            r = requests.post(self._server_path + 'sample',
+#                              data=domt)
+#            r.raise_for_status()
 
+    @property
+    def host(self):
+        return self.__host
+    
+    @host.setter
+    def host(self, new_host):
+        self.__host = new_host
```

Comment:
If there's no logic here, probably not worth having properties for this (as the getter/setter are identical to just accessing '.__host'

Also, probably not necessary to munge the name with double-underscore, single would suffice and not make it a pain to poke during debugging